R/mod_decision_automation_utils.R

```diff
@@ -5,7 +5,9 @@ assign_decisions <- function(decision_list, package) {
   score <- get_pkg_info(package)$score
   decision <- paste0(names(decision_list)[purrr::map_lgl(decision_list, ~ .x[1] < score && score <= .x[2])], "")
   if (decision != "") {
-    dbUpdate(glue::glue("UPDATE package SET decision = '{decision}' WHERE name = '{package}'"))
+    dbUpdate(glue::glue("UPDATE package SET decision = '{decision}',
+                        decision_by = 'auto_assign', decision_date = '{Sys.Date()}'
+                        WHERE name = '{package}'"))
```

Comment:
Just thinking of alternative names - perhaps "Auto Decision"? If we stick with "auto_assign", I think you could get rid of the underscore, use proper case, and add an "-ed" at the end to make it past tense. So the result would be: "Auto Assigned".

But if the column name changes to "Decision Assigned By" you could get rid of the word "assign" in "Auto Assigned". Instead, maybe it could just say "Automation".

Thoughts?